homeassistant/components/overkiz/water_heater_entities/atlantic_domestic_hot_water_production_mlb_component.py

```diff
@@ -158,10 +158,25 @@ async def async_set_operation_mode(self, operation_mode: str) -> None:
             )
 
     async def async_turn_away_mode_on(self) -> None:
-        """Turn away mode on."""
+        """Turn away mode on. This requires the start date and the end date to be also set."""
+        now_date = cast(
+            dict,
+            self.executor.select_state(OverkizState.CORE_DATETIME),
+        )
         await self.executor.async_execute_command(
-            OverkizCommand.SET_ABSENCE_MODE, OverkizCommandParam.ON
+            OverkizCommand.SET_ABSENCE_MODE,
+            OverkizCommandParam.PROG,
+            refresh_afterwards=False,
         )
+        await self.executor.async_execute_command(
+            OverkizCommand.SET_ABSENCE_START_DATE, now_date, refresh_afterwards=False
+        )
+        now_date["year"] = now_date["year"] + 1
+        await self.executor.async_execute_command(
```

Comment:
Can you add some comments to this function? Especially around why we bump the year by one, due to limitations in HA + Overkiz. This is not standard behavior, so good to document.

Reply:
I didn't find a suitable place to leave the comments. Can I just leave them in the code?

Reply:
Yes, you can add comments in the code or in the function definition. You will see some examples in other Overkiz files.